core-web/libs/dotcms-webcomponents/src/elements/dot-contentlet-thumbnail/dot-contentlet-thumbnail.tsx

```diff
@@ -18,8 +18,10 @@ export class DotContentletThumbnail {
     @Prop({ reflect: true })
     iconSize = '';
 
+    // JSP elements need the image to be rendered as a background image because of the way they are styled
+    // New elements should use the default image tag for accessibility
     @Prop({ reflect: true })
-    cover = true;
+    backgroundImage = false;
```

Comment:
Binary field dont have tests right?

Reply:
I updated the tests for the `web-component`